.github/workflows/hardware-long.yml

```diff
@@ -2,6 +2,10 @@ name: Hardware Long Tests
 
 # Run long tests once nightly, at 00:00
 on:
+  push:
```

Comment:
This test takes ~15 minutes, sometimes more. Do we want to run it on every merge?

Reply:
Woops - good catch. I think just release branches.

Reply:
Schedule should be against tip of tree

Reply:
Fixed. Yeah, I didn't realize that it only runs against the default branch.